### PR TITLE
PA TANF Age Eligibility

### LINKED ISSUE #2385: Pennsylvania TANF Age Eligibility

### PR DESCRIPTION
**Pull request by itsmexw**
Fixes #2385

### CODE REVIEW

File: policyengine_us/parameters/gov/states/pa/tanf/cash_assistance/age_eligibility.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description: Pennsylvania TANF age eligibility for child
+values:
+  2023-01-01: 18
+metadata:
+  unit: years
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/pa/tanf/cash_assistance/age_eligibility.yaml:5]:
```suggestion
  unit: year
```

File: policyengine_us/variables/gov/states/pa/tanf/cash_assistance/pa_tanf_age_eligible.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        p = parameters(period).gov.states.pa.tanf.cash_assistance
+        age = person("age", period)
+        # Get full time students
+        student_eligible = person("is_full_time_student", period) & (
+            age == p.age_eligibility
+        )
+
+        # Get age
+        is_eligible_age = age < p.age_eligibility
+        eligible = spm_unit.any(is_eligible_age | student_eligible)
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/pa/tanf/cash_assistance/pa_tanf_age_eligible.py:22]:
```suggestion
        return spm_unit.any(is_eligible_age | student_eligible)
```

File: policyengine_us/variables/gov/states/pa/tanf/cash_assistance/pa_tanf_age_eligible.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+        return eligible
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/pa/tanf/cash_assistance/pa_tanf_age_eligible.py:24]:
```suggestion
```

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: run `git pull upstream master`

File: policyengine_us/parameters/gov/states/pa/tanf/cash_assistance/age_eligibility.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+description: Pennsylvania provides the TANF supplement for children under this age eligibility.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/pa/tanf/cash_assistance/age_eligibility.yaml:1]:
```suggestion
description: Pennsylvania limits TANF eligibility to people below this age, or students exactly this age.
```

File: policyengine_us/parameters/gov/states/pa/tanf/cash_assistance/age_eligibility.yaml
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/pa/tanf/cash_assistance/age_eligibility.yaml:1]: Rename `age_limit.yaml`

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**